test-suite/djinni/extended_record.djinni

```diff
@@ -0,0 +1,5 @@
+
+# Extended record
+extended_record = record +c {
+    foo: bool;
+}
```

Comment:
If you want to exercise a case where the forward-decl is actually required, you could add a constant of type extended_record here.